test/php/library/Icinga/Chart/GraphChartTest.php

```diff
@@ -7,9 +7,19 @@
 use DOMDocument;
 use Icinga\Chart\GridChart;
 use Icinga\Test\BaseTestCase;
+use Icinga\Util\Csp;
+use Icinga\Web\Session;
+use Tests\Icinga\Lib\FakeSession;
 
 class GraphChartTest extends BaseTestCase
 {
+    public function setUp(): void
+    {
+        parent::setUp();
+        Session::create(new FakeSession());
+        Csp::createNonces();
+    }
```

Comment:
Since you're no longer mocking the session class, please move this initialisation back to the `BaseTestCase` class and ...

Reply:
As mentioned by @nilmerg in this [comment](https://github.com/Icinga/icingaweb2/pull/5060#discussion_r1304028255) the session will still be mocked in `FormTest`. Hence I believe this cannot be moved to `BaseTestCase`